src/engine/meta/meta_blks_mgr.cpp

```diff
@@ -399,6 +400,14 @@ void MetaBlkMgr::register_handler(const meta_sub_type type, const meta_blk_found
     HS_LOG(INFO, metablk, "[type={}] registered with do_crc: {}", type, do_crc);
 }
 
+void MetaBlkMgr::create_sobject(meta_sub_type type) {
+    if(m_sub_info[type].meta_bids.size()==1) {
```

Comment:
Have you run clang-format on your editer? It seems it is not doing format for you, normally it should be:
if (m_sub_info[type].meta_bids.size() = 1) // << with space in between